README.md

```diff
@@ -32,42 +31,132 @@ Everything is co-ordinated through github issues.  The current development proce
+
+- the installation of depencies
+- running the application
+
+### Installing dependencies
+
+To install dependencies, simply run `flutter pub get` in your terminal.
+
+### Developing the app
+
+#### iOS
+
+Initialise an instance of an iOS Simulator or connect physical iOS device, navigate to the cloned repo and run from root:
+
+```sh
+# from terminal
+
+open -a simulator
+
+cd desktop/muhnee-app
+flutter run -d "iPhone"
+```
+
+#### Android
+
+##### Keychain Certificate Setup
+
+1. Download the `google-services.json` configuration file from the Firebase Project and move it to the directory `<PROJECT-ROOT>/android/app`
+2. Get the Android Keychain by:
+
+   ```
+   keytool -list -v -keystore ~/.android/debug.keystore -alias androiddebugkey -storepass android -keypass a
+   ndroid
+   ```
+
+3. Copy either the SHA1 or the SHA256 certificate fingerprint onto the Android App on [Firebase Console]()
+
+##### Development
+
+If you have a device plugged in just run:
+
+```
+flutter run
+```
+
+Otherwise if you have setup [Android Studio](https://developer.android.com/studio) and/or the Android Software Development Kit (SDK) you can run it over a virtual Android emulator using the Android Debug Bridge (ADB). Just simply run:
+
+```
+flutter run adb
+```
+
+##### Releasing
+
+1. Create a Keystore
+
+   If you have an existing keystore, skip to the next step. If not,create one by running the following at the command line:
+
+   On Mac/Linux, use the following command:
+
+   ```sh
+   keytool -genkey -v -keystore ~/key.jks -keyalg RSA -keysize 2048 -validity 10000 -alias key
+   ```
+
+   On Windows, use the following command:
+
+   ```sh
+   keytool -genkey -v -keystore c:/Users/USER_NAME/key.jks -storetype JKS -keyalg RSA -keysize 2048 -validity 10000
+   ```
+
+2. Create a file named <app dir>/android/key.properties that contains a reference to your keystore:
+
+   ```
+   storePassword=<password from previous step>
+   keyPassword=<password from previous step>
+   keyAlias=key
+   storeFile=<location of the key store file, such as /Users/<user name>/key.jks>
+   ```
+
+3. You will also need to update the SHA1/SHA256 Certificate Fingerprint in Firebase Console with the signing certificate.
+
+   To get the certificate run:
 
-Currently all data will be stored inside the app, with no CMS or web services.  The app should have minimal logic, and instead just focuses on providing important information in a mobile friendly format.
+   macOS/Linux:
 
-## Getting Started
+   ```sh
+   keytool -list -v -alias key -keystore ~/key.jks
+   ```
 
-*TODO:  Flutter specific information*
+   Windows:
 
-For help getting started with Flutter, view googles
-[online documentation](https://flutter.dev/docs), which offers tutorials,
-samples, guidance on mobile development, and a full API reference.
+   ```sh
+   keytool -list -v -alias key -keystore c:/Users/USER_NAME/key.jks
```

Comment:
Also no need for any docs about keystore and signing. Well have a private doc for admins about how this is setup in CI

Reply:
Done as part of the removal of the releasing section!